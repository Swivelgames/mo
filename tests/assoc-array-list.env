declare -a navItems nonFunctional
declare -A homeLink otherLink

homeLink[title]="Home"
homeLink[url]="https://www.example.com/home"

otherLink[title]="Other Link"
otherLink[url]="https://www.example.com/other-page"

navItems=(homeLink otherLink)
stringArray=("homeLink" "otherLink")

