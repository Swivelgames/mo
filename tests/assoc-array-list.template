# Links
{{#navItems}}
- [{{title}}]({{url}})
{{/navItems}}

# Standard List
{{#stringArray}}
- {{.}}
{{/stringArray}}

