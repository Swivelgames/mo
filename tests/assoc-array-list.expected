# Links
- [Home](https://www.example.com/home)
- [Other Link](https://www.example.com/other-page)

# Standard List
- homeLink
- otherLink

